forge-ai/src/main/java/forge/ai/simulation/GameSimulator.java

```diff
@@ -158,9 +158,9 @@ public Score simulateSpellAbility(SpellAbility origSa, boolean resolve) {
     }
     public Score simulateSpellAbility(SpellAbility origSa, GameStateEvaluator eval, boolean resolve) {
         SpellAbility sa;
-        if (origSa instanceof LandAbility) {
+        if (origSa.isLandAbility()) {
             Card hostCard = (Card) copier.find(origSa.getHostCard());
-            if (!aiPlayer.playLand(hostCard, false)) {
+            if (!aiPlayer.playLand(hostCard, false, origSa)) {
```

Comment:
Simulation now refuses to play lands with _Serra Paragon_ (given no other options)

Reply:
Ugh, can you debug it what the right chante should be?

Reply:
it wasn't failing there, but for some reason your new LandAbility had `ValidAfterStack$`